.gitignore

```diff
@@ -1,5 +1,6 @@
 database.db
 *pyc
+books/migrations/*
```

Comment:
A note: migration files *are* supposed to be part of the distribution (even if they look rather ugly) and needed for the complete package, in order to allow users to update the DB schema and data seamlessly when a new version modifies the models (something [like this](http://stackoverflow.com/questions/28035119/should-i-be-adding-the-django-migration-files-in-the-gitignore-file)). I hope the line was meant to prevent accidental inclusion or problems when several developers are working on features that modify models - leaving it there for that reason is fine, and maybe save some trouble indeed.